ui/start_ui.py

```diff
@@ -20,6 +20,52 @@
 import sys
 
 sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+import importlib, pip
+
+def fix_package_name(package):
+    a = package.split('>')[0]
+    a = a.split('<')[0]
+    b = a.split('=')[0]
+    
+    package_name_map = {
+    }
+    
+    if b in package_name_map:
+        b = package_name_map[b]
+    return b
+
+def check_availability_and_install(package_or_list, verbose=1):
+    def actual_func(package):
+        pip_name = fix_package_name(package)
+        try:
+            if pip_name == 'gradio':
+                import gradio
+                a = gradio.Progress()
+            else:
+                return importlib.import_module(pip_name)
+        except ImportError:
+            print(f"automatically install for {package}")
+            pip.main(['install', '-q', package])
+            #importlib.import_module(pip_name)
+            
+    if isinstance(package_or_list, list):
+        if verbose == 1 and len(package_or_list) > 0:        
+            print(f"check_availability_and_install {package_or_list}")
+        for pkg in package_or_list:
+            actual_func(pkg)
+    elif isinstance(package_or_list, str):
+        if verbose == 1 and package_or_list != "":        
+            print(f"check_availability_and_install {package_or_list}")
+        actual_func(package_or_list)
+    else:
+        raise ValueError(f"{package_or_list} with type of {type(package_or_list)} is not supported.")
+    
+check_availability_and_install(['gradio==3.36.1', 'gradio_client==0.7.3', 'langchain==0.1.4', 'langchain-community==0.0.16', 'lz4', 'sentence-transformers==2.2.2', 'pyrecdp'])
```

Comment:
There is a script in https://github.com/intel/llm-on-ray/pull/101 to install the UI environment, maybe we don't need to check it in code anymore. And we have to use `pip install 'git+https://github.com/intel/e2eAIOK.git#egg=pyrecdp&subdirectory=RecDP'` to install pyrecdp, `pip install pyrecdp` does not contain the latest code and will change the ray version.
```
Collecting ray==2.7.1 (from pyrecdp)
  Using cached ray-2.7.1-cp39-cp39-manylinux2014_x86_64.whl.metadata (13 kB)
```